spotseeker_server/migrations/0001_initial.py

```diff
@@ -2,28 +2,22 @@
 from __future__ import unicode_literals
 
 from django.db import models, migrations
+import django.contrib.auth.models
+import django.contrib.auth.validators
 import re
 from django.conf import settings
 import django.core.validators
+import django.utils.timezone
 
 
 class Migration(migrations.Migration):
```

Comment:
Afraid this might break on a database that has already run these migrations (when the later ones are run.) Really need to get a back up of the data and do some thorough testing of migrations against a test db